server/data/wikidata/queries/editions_reverse_claims.coffee

```diff
@@ -7,7 +7,10 @@ module.exports =
       ?item wdt:#{pid} wd:#{qid} .
       ?item wdt:P31 wd:Q3331189 .
       # Filter-out entities tagged as both work and edition
-      FILTER NOT EXISTS { ?item wdt:P31 wd:Q571 }
+      FILTER NOT EXISTS {
+        ?item wdt:P31 ?work_type
+        VALUES (?work_type) { (wd:Q571) (wd:Q47461344) (wd:Q7725634) (wd:Q2831984) (wd:Q1004) (wd:Q1760610) (wd:Q8274) } .
```

Comment:
A usual pattern is to define a variable before using it, but here `?work_type` is defined after.

Reply:
done in 04397e42